pkg/kudoctl/packages/resolver/resolver.go

```diff
@@ -49,32 +49,62 @@ func NewInClusterResolver(c *kudo.Client, ns string) Resolver {
 // For local access there is a need to provide absolute or relative path as part of the name argument. `cassandra` without a path
 // component will resolve to the remote repo.  `./cassandra` will resolve to a folder which is expected to have the operator structure on the filesystem.
 // `../folder/cassandra.tgz` will resolve to the cassandra package tarball on the filesystem.
-func (m *PackageResolver) Resolve(name string, appVersion string, operatorVersion string) (p *packages.Resources, err error) {
+func (m *PackageResolver) Resolve(name string, appVersion string, operatorVersion string) (*packages.FancyResources, error) {
 
-	// Local files/folder have priority
-	_, err = m.local.fs.Stat(name)
-	// force local operators usage to be either absolute or express a relative path
-	// or put another way, a name can NOT be mistaken to be the name of a local folder
-	if filepath.Base(name) != name && err == nil {
-		var abs string
-		abs, err = filepath.Abs(name)
-		if err != nil {
-			return nil, err
-		}
+	clog.V(1).Printf("determining package type of %v", name)
```

Comment:
Nit: We have a mix of V(2) and V(3) logs in this file, why is this a V(1) log in comparison to the other logs in the file? IMO, this log line really help as addition to the existing logs, but not standalone, i.e should be a V(2) log.

Reply:
agreed.